tests/console/cleanup_qam_testrepos.pm

```diff
@@ -30,7 +30,7 @@ sub get_repo_aliases() {
 
 sub run {
     my $self = shift;
-    return if (script_run('zypper -n ref') == 0);    # If all repos are OK we don't have to do anything
+    return if (script_retry('zypper -n ref', timeout => (1200 ? is_public_cloud : 300), retry => 3, delay => 30) == 0); # If all repos are OK we don't have to do anything
```

Comment:
lovely ternary operator, thanks! :) 
Interesting way to use it, btw. I would have done `(is_public_cloud ? 1200 : 300)`

Reply:
I'm apparently in dire need of some proper coffee today :coffee:

Reply:
I'd also do `if if_public_cloud then 1200 else 300` :smile: